server/lib/pencil_space_server_web/controllers/game_controller.ex

```diff
@@ -4,33 +4,47 @@ defmodule PencilSpaceServerWeb.GameController do
   """
 
   use PencilSpaceServerWeb, :controller
-  alias PencilSpaceServer.{GameMonitor}
+  alias PencilSpaceServer.{Game}
+  import PencilSpaceServerWeb.ControllerHelpers
 
-  def create(conn, _params) do
-    game_id = Nanoid.generate()
+  def create(
+        conn,
+        %{
+          "host" => %{
+            "id" => _id,
+            "name" => _name,
+            "avatar" => _avatar
+          }
+        } = host
+      ) do
 
-    case GameMonitor.start(game_id) do
-      {:ok, _pid} ->
-        conn
-        |> put_status(200)
-        |> json(%{id: game_id})
+    case Game.start do
+      {:ok, name} ->
+        Game.update(name, host)
```

Comment:
If host is guaranteed to be present at the time of Game gen_server creation, maybe we can include this in the `Monitor`'s init function instead.

Currently we are waiting to create a GenServer, and once it is created we are sending it a message again.

This is okay, but not optimal in the strictest sense.